convex/accessControl.ts

```
@@ -0,0 +1,565 @@
+      });
+
+      let accessResult: AccessResult;
+
+      // Check resource-specific access
+      switch (resourceType) {
+        case "notebook":
+          if (!args.notebookId && !args.url) {
+            throw new ConvexError("Notebook identifier required");
+          }
+          
+          let notebookId = args.notebookId;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)